### PR TITLE
support for custom 'internal_repo' path

### PR DESCRIPTION
**Pull request by irosenzw**
Adding support for creating an 'internal_repo' folder that isn't in the default location: "deployment-.../default/internal_repo/"

Signed-off-by: Ido Rosenzwig <irosenzw@redhat.com>

### CODE REVIEW

**Comment by eedri** [comment]
ci test please

File: lago/prefix.py [hunk 1]
```diff
@@ -362,16 +369,14 @@ def _register_preallocated_ips(self, conf):
...
                     raise RuntimeError(
-                        'IP %s was to several domains: %s %s' % (
-                            nic['ip'],
-                            dom_name,
-                            ' '.join(conflict_list),
-                        ),
+                        'IP %s was to several domains: %s %s' %
+                        (nic['ip'],
+                         dom_name,
+                         ' '.join(conflict_list), ),
```
Comment (david-caro) [inline lago/prefix.py]: This changes are not related to the commit, try not to mix indent/style changes with the commit changes please.

File: lago/ssh.py [hunk 2]
```diff
@@ -354,7 +359,7 @@ def get_ssh_client(
                 )
 
             ssh_tries -= 1
-            time.sleep(1)
+            time.sleep(5)
```
Comment (david-caro) [inline lago/ssh.py]: This change is totally alien to the pr too, pleas split into another pr (or explain the relation to the custom internal repo)

File: lago/prefix.py [hunk 3]
```diff
@@ -104,6 +104,13 @@ def __init__(self, prefix):
         self.paths = paths.Paths(self._prefix)
         self._virt_env = None
         self._metadata = None
+        self._internal_repo = None
+
+    def _get_internal_repo(self):
+        return self._internal_repo
+
+    def _set_internal_repo(self, path):
```
Comment (david-caro) [inline lago/prefix.py]: There's no need for getters and setters, this is python, use prefix.internal_repo instead (just make it public by convention)

**Comment by david-caro** [comment]
The style changes you introduced are the ones that are breaking the build, make sure to run yapf on the code you change to reformat to the same style that it's used in the rest of the code:

http://lago.readthedocs.io/en/stable/Env_Setup.html#style-formatting

btw. we are right now pinned on yapf==0.7.1, you can see all the test deps in the test-requires.txt.

**Comment by mykaul** [comment]
@irosenzw  - can you reply to the comments, rebase and repost?
